recipes/cnsr/build.sh

```diff
@@ -0,0 +1,2 @@
+$PYTHON -m pip install . -vv
+install_cnsr_launcher_data
```

Comment:
I think it might be safer to just invoke the module directly like:

```suggestion
${PYTHON} -m cnsr.install
```

Reply:
That said, that function is only copying files to the install location... shouldn't those be part of the MANIFEST data? 🤔 By tying this at build time you are imposing a dependency on the Jupyter stack tree structure, so I am not sure if this is going to cause trouble in the future... Can you provide more details on what [this function](https://github.com/ssciwr/cnsr/blob/d2a9842687a257e7643a351b78e2304c30d3bb20/cnsr/install.py#L32) is supposed to do? Thanks!

Reply:
The function copies a configuration for [jupyter_app_launcher](https://github.com/trungleduc/jupyter_app_launcher) along with some notebooks into the Jupyter data directory (which is in `$PREFIX/share`). To my knowledge, `MANIFEST.in` gives me only control of what is installed into the Python site packages directory. Unfortunately, jupyter_app_launcher requires me to use relative paths, so the notebooks need to be in the Jupyter data directory as well. The final goal of the project is that `conda install -c conda-forge cnsr` will give a JupyterLab starting screen with shortcuts for all analysis notebooks relevant to users of the [CNSR lab facility](https://cnsr.uni-heidelberg.de/).

Reply:
Hm, that feels a bit hacky, and after seeing this warning in the logs (see below), I wonder if it would be too much trouble implementing in that way:

```
/home/conda/staged-recipes/build_artifacts/cnsr_1695108619684/work/cnsr/install.py:37: DeprecationWarning: Jupyter is migrating its paths to use standard platformdirs
given by the platformdirs library.  To remove this warning and
see the appropriate new directories, set the environment variable
`JUPYTER_PLATFORM_DIRS=1` and then run `jupyter --paths`.
The use of platformdirs will be the default in `jupyter_core` v6
  from jupyter_core import paths
```

Reply:
I reimplemented the path fiddling part and used the same approach that e.g. Jupyter kernels use. See for example here: https://github.com/conda-forge/bash_kernel-feedstock/blob/main/recipe/meta.yaml#L17

Reply:
The issue I see is that this will create a generic `$PREFIX/share/jupyter_app_launcher/config.yaml`, silently overwriting existing configs if they are there. I see that you already posted https://github.com/trungleduc/jupyter_app_launcher/issues/36 so I know you are aware of this caveat (thanks for doing that btw!). All you could do right now is to error out if there's a collision (or try to merge the contents), but that's a `post-link.sh` or `activate.sh` business, I'd say.

Anyway, sorry for being annoying! This is good to go imo.